inst/plumber/v1/endpoints.R

```diff
@@ -241,6 +241,14 @@ function(req) {
   params$lkup <- lkups$versions_paths[[params$version]]
   params$format <- NULL
   params$version <- NULL
+
+  # Define default arguments
+  if (is.null(params$country))
+    params$country <- "all"
+  if (is.null(params$year))
+    params$year <- "all"
+
+  # Parallel processing for slow requests
   if (params$country == "all" && params$year == "all") {
     out <- promises::future_promise({
```

Comment:
So this is what's causing the call to fail... Not sure what's tje nest way to handle this. So far, missing query parameter where handled through the corresponding  function's default argument. Now we are introducing another layer of logic by coercing some parameters... I don't feel totally comfortable with that. Even though I cannot exactly put my finger on what bothers me (And I don't have a better solution either...)